tardis/io/tests/data/tardis_model_format.csv

```diff
@@ -0,0 +1,12 @@
+0.976 day
+velocity densities electron_densities temperature C O Mg Si Ni56 Ni58
+871.66905 4.2537191E-09 259538070000000 7.6395577 0 0 0 0.6 0.4 0
```

Comment:
electron_densities - should be float. 
and add one more line with units for the different quantities. with 1 for dimensionless units (such as abundance fractions)>